### PR TITLE
Feat/improve constants

### PR DESCRIPTION
**Pull request by marcosalves-bitg**
This PR adds some improvements to http messages and status constants.
They're now enums. And we now have a function that returns a default response object with a correct name is sent

### CODE REVIEW

**Review by Viserion77 — DISMISSED** [review]
LGTM

**Review by Viserion77 — APPROVED** [review]
IATB

**Review by ixpd — DISMISSED** [review]
LGTM

**Review by GabrielPedrotti — DISMISSED** [review]
LGTM

File: src/http/enums.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  forbidden = 403,
+  notFound = 404,
+  userNotFound = 404,
+  integrationError = 406,
+  teaPot = 418,
+  unprocessableEntity = 422,
+  internalError = 500,
+}
+
+export enum HttpNames {
```
Comment (alanneves) [inline src/http/enums.ts:33]: That makes sense?

**Review by alanneves — CHANGES_REQUESTED** [review]
That makes sense?

File: src/http/enums.ts (hunk 1 above)
Comment (alanneves) [inline src/http/enums.ts:33]: That makes sense?

**Review by alanneves — CHANGES_REQUESTED** [review]

**Review by Viserion77 — APPROVED** [review]
ELGETEEME

**Review by alanneves — APPROVED** [review]
LGTM

**Pull request merged**